src/docs/patterns/account-menu.mdx

```diff
@@ -0,0 +1,190 @@
+    regularWidth
+    caption={[
+        `1. Place the top account-related actions at the bottom of the drawer.`,
+        `2. Avoid overcrowding the drawer by including too many user actions.`,
+    ]}
+/>
+
+This approach is most effective when there are relatively few account-related actions and is also preferred on mobile where there is limited space in the app bar.
+
+If you are using a navigation rail, you may place account settings links in the rail, but you will not be able to show specific details about the user. Consider using the [User Menu](#via-user-menu) in this situation or if you are using a primary navigation mechanism other than a drawer.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Drawer Navigation'}
+        url={'/patterns/navigation#drawer-navigation'}
+        description={`Follow PX Blue's guideline for using a Drawer.`}
+        avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+    />
+    <Divider />
+</React.Fragment>
+
+# In a User Menu
+
+<ImageGrid
+    images={[UserMenuContents]}
+    regularWidth
+    caption={[
+        `1. On desktop, user menu contents appear in a dropdown menu below the avatar.`,
+        `2. On mobile devices, the contents should be presented in a bottom sheet.`,
+    ]}
+/>
+
+Another way to present user information and actions is with the user menu. The user menu appears as an Avatar in the [App Bar](/patterns/appbar) which opens a menu with the user-related content (see the [contents](#contents) section below).
+
+This approach is most effective if you have more than three account-related actions, or if you want to present more detailed information about a user since it avoids cluttering the primary navigation drawer.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Bottom Sheet'}
+        url={'/patterns/overlay#bottom-sheet'}
+        description={`Follow PX Blue's guideline for using a Bottom Sheet.`}
+        avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+    />
+    <Divider />
+</React.Fragment>
+
+# Contents
+
+Whether you are using the drawer or a dedicated user menu component, the content should be very similar.
+
+## Avatar
+
+<ImageGrid
+    images={[AvatarVariants]}
+    regularWidth
+    caption={[
+        `1. Generic avatar icon for use with shared or anonymous accounts`,
+        `2. Basic avatar showing user's initials.`,
+        `3. Avatar showing a user-selected profile picture.`,
+        `4. Avatar with a notification indicator.`,
+    ]}
+/>
+
+In the header, you should present an avatar that identifies the user along with basic information about them, such as their name, username, email, job title, etc. The avatar can take several forms. Usually, it will display a user-selected profile picture. If your application does not support user-selected images, you can default to displaying the user's initials or a generic [person icon](/style/iconography?icon=Person&isMaterial=true).
```

Comment:
It confused me when you start this section with images of app bars, and then say "in the header"... it confused me which header you are referring to, since appbars are frequently referred to as "headers".